cantools/subparsers/plot.py

```diff
@@ -51,14 +51,22 @@
 try:
     from matplotlib import pyplot as plt
 except ImportError:
-    print("matplotlib package not installed. Required for producing plots.")
+    plt = None
 
 from .. import database
+from .. import errors
 
 
-plt.rcParams["date.autoformatter.hour"] = "%H:%M"
-plt.rcParams["date.autoformatter.minute"] = "%H:%M"
-plt.rcParams["date.autoformatter.microsecond"] = "%H:%M:%S.%f"
+class MatplotlibNotInstalledError(errors.Error):
+    def __init__(self):
+        super().__init__("The matplotlib package not installed and is required "
+                         "for producing plots.")
```

Comment:
You forgot the blank line between line 60 and 61.

Reply:
You should get an autoformatter...